prisma/schema.prisma

```diff
@@ -44,14 +44,15 @@ model EventType {
   Schedule                Schedule[]
   price                   Int                    @default(0)
   currency                String                 @default("usd")
+
   @@unique([userId, slug])
 }
 
 model Credential {
   id     Int    @id @default(autoincrement())
   type   String
   key    Json
-  user   User?  @relation(fields: [userId], references: [id])
+  user   User?  @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
```

Comment:
This means that a user will automatically delete any of the user's credentials

Reply:
makes sense